bench/state_transition/process_epoch.zig

```diff
@@ -0,0 +1,491 @@
+        defer {
+            cloned.deinit();
+            self.allocator.destroy(cloned);
+        }
+        const epoch_cache = cloned.getEpochCache();
+        const effective_balance_increments = epoch_cache.getEffectiveBalanceIncrements();
+        state_transition.processProposerLookahead.processProposerLookahead(self.allocator, cloned.state, &effective_balance_increments) catch return;
+    }
+};
+
+pub fn main() !void {
+    const allocator = std.heap.page_allocator;
+    const stdout = std.io.getStdOut().writer();
+
+    try stdout.print("Loading state from SSZ file...\n", .{});
+
+    // Load state from SSZ
+    const state_file = try std.fs.cwd().openFile("bench/state_transition/state.ssz", .{});
+    defer state_file.close();
+    const state_bytes = try state_file.readToEndAlloc(allocator, 10_000_000_000);
+    defer allocator.free(state_bytes);
+
+    try stdout.print("State file loaded: {} bytes\n", .{state_bytes.len});
+
+    const fulu_state = try allocator.create(BeaconState.Type);
+    fulu_state.* = BeaconState.default_value;
+    try BeaconState.deserializeFromBytes(allocator, state_bytes, fulu_state);
+
+    try stdout.print("State deserialized: slot={}, validators={}\n", .{ fulu_state.slot, fulu_state.validators.items.len });
+
+    // Create beacon state wrapper
+    const beacon_state = try allocator.create(BeaconStateAllForks);
+    beacon_state.* = .{ .fulu = fulu_state };
+
+    // Create cached state with mainnet config
+    const chain_config = config.mainnet_chain_config;
+    const beacon_config = try config.BeaconConfig.init(allocator, chain_config, beacon_state.genesisValidatorsRoot());
+
+    const pubkey_index_map = try PubkeyIndexMap.init(allocator);
+    const index_pubkey_cache = try allocator.create(state_transition.Index2PubkeyCache);
+    index_pubkey_cache.* = state_transition.Index2PubkeyCache.init(allocator);
+
+    try state_transition.syncPubkeys(beacon_state.validators().items, pubkey_index_map, index_pubkey_cache);
+
+    const immutable_data = state_transition.EpochCacheImmutableData{
+        .config = beacon_config,
+        .index_to_pubkey = index_pubkey_cache,
+        .pubkey_to_index = pubkey_index_map,
+    };
+
+    const cached_state = try CachedBeaconStateAllForks.createCachedBeaconState(allocator, beacon_state, immutable_data, .{
+        .skip_sync_committee_cache = false,
+        .skip_sync_pubkeys = false,
+    });
+
+    try stdout.print("Cached state created at slot {}\n", .{cached_state.state.slot()});
+
+    try stdout.print("\nStarting process_epoch benchmarks...\n\n", .{});
+
+    var bench = zbench.Benchmark.init(allocator, .{});
+    defer bench.deinit();
+
+    const process_justification_bench = ProcessJustificationAndFinalizationBench{
+        .allocator = allocator,
+        .cached_state = cached_state,
+    };
+    try bench.addParam("justification_finalization", &process_justification_bench, .{});
```

Comment:
let's try to be accurate with naming.

```suggestion
    const process_justification_and_finalization_bench = ProcessJustificationAndFinalizationBench{
        .allocator = allocator,
        .cached_state = cached_state,
    };
    try bench.addParam("justification_and_finalization", &process_justification_and_finalization_bench, .{});
```